<launch>
	<node pkg="couch_controller" type="couch_controller_node" name="couch_controller_node" output="screen">
		<param name="ip_addr" value="192.168.1.185" />
		
		<param name="publish_hz" value="100" />
		<param name="cmdvel_timeout" value="0.5" /> <!-- seconds -->

		<param name="max_linear" value="5.0" />
		<param name="max_angular" value="5.0" />

		<param name="kf" value="0.2" />
		<param name="kp" value="0.1" />
		<param name="ki" value="0.06" />
		<param name="kd" value="0.00" />
		<param name="kMaxAccel" value="4.0" />
	</node>
</launch>

